engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -3246,9 +3247,9 @@ Common::Point Myst::towerRotationMapComputeCoords(uint16 angle) {
 	Common::Point end;
 
 	// Polar to rect coords
-	double radians = angle * M_PI / 180.0;
-	end.x = (int16)(_towerRotationCenter.x + cos(radians) * 310.0);
-	end.y = (int16)(_towerRotationCenter.y + sin(radians) * 310.0);
+	float radians = Common::deg2rad<uint16,float>(angle);
+	end.x = (int16)(_towerRotationCenter.x + cos(radians) * 310.0f);
+	end.y = (int16)(_towerRotationCenter.y + sin(radians) * 310.0f);
```

Comment:
What is the reason for changing the code to use simple precision here? Because of the cast to int16 at the end? Due to the change of precision, this might in some edge case end up casting to a different integral value. This might be fine, but @bgK might want to check this change.

Reply:
It's fine. Using double precision is not needed here.